lk2nd/device/dts/msm8994/msm8994-huawei-angler.dts

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	model = "Huawei Nexus 6P";
```

Comment:
I think we decided to not specify model in / unless bootloader explicitly wants it

Reply:
Oh, okay, I''ll test with it removed, if it boots I'll add a follow-up commit